instagram-dl.user.js

```diff
@@ -532,6 +532,11 @@
     }
 
     function findPostName(articleNode) {
+        let imgNoCanvas = articleNode.querySelector('article section + * a[href^="/"][href$="/"]');

```

Comment:
It'd be nice to see a comment here indicating what element it's supposed to be selecting.

Reply:
updated!